README.md

```diff
@@ -1,59 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TODO List
 
-> One paragraph statement about the project.
+>"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete..
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
-
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- JavaScript
+- Webpacks
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
+[Live Demo Link](https://shadyshawkat.github.io/TODO-List/dist)
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1- Navigate to the button on the top right green "code" button.
+2- In the drop down menu choose "Download Zip".
+3- After extracting the files, you'll have my project on your local machine.
```

Comment:
- [ ] I would recommend writing all the clear steps and commands which is necessary to run the project locally. Like `npm i`, `npm start`.